app/helpers/lookup_keys_helper.rb

```diff
@@ -36,9 +36,7 @@ def show_puppet_class(f)
     if params["puppetclass_id"]
       select_f f, :puppetclass_id, [Puppetclass.find(params["puppetclass_id"])], :id, :to_label, {}, {:label => _("Puppet class"), :disabled => true}
     elsif f.object.puppet? && f.object.param_class
-      field(f, :puppetclass_id, :label => _('Puppet Class')) do
-        content_tag(:input, nil, :value => f.object.param_class, :type => 'text', :disabled => true)
-      end
+      text_f(f, :puppetclass_id, :label => _('Puppet Class'), :value => f.object.param_class, :disabled => true)
```

Comment:
:+1: When testing I noticed that specific field looked odd, so fixed it on the way.